### ISSUE TITLE
[Bug] System DBus not available by default on Flatpak install

### ISSUE THREAD

**Issue by ppom0** [KRTirtho/spotube#359]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I run the app via Flatpak, there is this error in the logs:
```console
$ flatpak run com.github.KRTirtho.Spotube
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: SocketException: Connection failed (OS Error: No such file or directory, errno = 2), address = /var/run/dbus/system_bus_socket, port = 0
#0      _NativeSocket.startConnect (dart:io-patch/socket_patch.dart:682)
#1      _NativeSocket.connect (dart:io-patch/socket_patch.dart:948)
#2      _RawSocket.connect (dart:io-patch/socket_patch.dart:1815)
#3      RawSocket.connect (dart:io-patch/socket_patch.dart:21)
#4      DBusClient._openSocket (package:dbus/src/dbus_client.dart:767)
#5      DBusClient._connect (package:dbus/src/dbus_client.dart:787)
#6      DBusClient.callMethod (package:dbus/src/dbus_client.dart:607)
#7      DBusClient.getNameOwner (package:dbus/src/dbus_client.dart:448)
#8      DBusClient._findUniqueName (package:dbus/src/dbus_client.dart:624)
#9      DBusSignalStream._onListen (package:dbus/src/dbus_client.dart:131)
[...]

flutter: RangeError (byteOffset): Invalid value: Valid value range is empty: 0
```

### Expected Behavior

I don't know why the system DBus is needed, but it should be made accessible in the Flatpak sandbox.

### Steps To Reproduce

1. With flatpak
2. `flatpak run com.github.KRTirtho.Spotube`
3. See error up here ↑

### Spotube Version

v2.6.0

### Operating System

Linux

### Linux Distribution (if applicable)

NixOS

### Operating System Version

NixOS 22.11

### Installation Source

Flathub (Flatpak)

### Additional information

I could make rid of this error by adding `--socket=system-bus` to the flatpak command:
`flatpak run --socket=system-bus com.github.KRTirtho.Spotube`

So I believe this socket should be added to the permissions in the flatpak packaging.

But maybe accessing the system DBus socket is useless, as I don't see which functionnalities are broken without `--socket=system-bus`. In which case you may remove this session opening in the code? (again, I don't know what it's for)

**Comment by KRTirtho** [comment]
This has been fixed already. Please check the latest version & close the issue

**Comment by KRTirtho** [comment]
Fixed in v3.1.1
